docs/using-scylla/cassandra-compatibility.rst

```diff
@@ -30,9 +30,10 @@ Interfaces
      - | Fully compatible with version 3.3.1, with additional features from later CQL versions (for example, :ref:`Duration type <durations>`).
        | Fully compatible with protocol v4, with additional features from v5.
      - More below
-   * - Thrift 
-     - Deprecated in ScyllaDB and Cassandra
-     - Support for the Thrift protocol is deprecated and will be dropped in future releases of ScyllaDB. 
+   * - Thrift
+     - Not supported anymore in ScyllaDB 5.5
```

Comment:
Again, is it really "5.5"?

Reply:
now. should be 6.0. as explained above. this document was updated before we cut 6.0. revised accordingly.